vertx-web-openapi/pom.xml

```diff
@@ -56,6 +56,11 @@
       <groupId>io.vertx</groupId>
       <artifactId>vertx-json-schema</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-openapi</artifactId>
+      <version>4.4.0-SNAPSHOT</version>
```

Comment:
We should not have a version, it needs to come from the stack, if it doesn't work it means we still need to add it the stack